maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.maps.layers
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.odk.collect.maps.databinding.OfflineMapLayerBinding
+import org.odk.collect.strings.localization.getLocalizedString
+
+class OfflineMapLayersAdapter(
+    private val layers: List<ReferenceLayer>,
+    private var selectedLayerId: String?,
+    private val onSelectedLayerChanged: (String?) -> Unit
+) : RecyclerView.Adapter<OfflineMapLayersAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val binding = OfflineMapLayerBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return ViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.binding.radioButton.setChecked(false)
```

Comment:
Right, I wanted to avoid creating a custom view for list elements but that would make testing easier. Let's wait until it has its final form (expandable with the delete button).